testshared/lib/fixture/my_space_fixture.dart

```diff
@@ -215,4 +215,50 @@ final recipients = [
 ];
 
 final fileInfo1 = FileInfo('fileInfo1', 'filePath1', 10);
-final fileInfo2 = FileInfo('fileInfo1', 'filePath1', 20);
\ No newline at end of file
+final fileInfo2 = FileInfo('fileInfo1', 'filePath1', 20);
+
+final sorter = Sorter(OrderScreen.mySpace, OrderBy.modificationDate, OrderType.descending);
+final sorter1 = Sorter(OrderScreen.mySpace, OrderBy.creationDate, OrderType.descending);
+final orderScreen = OrderScreen.mySpace;
+
+final documentSort1 = Document(
```

Comment:
Why do you need sorted document I don't get it ?

Reply:
Let me do unit tests for sort file

Reply:
Yes but thought sorted document was just a basic document

Reply:
Well, it looks like a basic document too. But I want to make sure the modificationDate, creationDate properties are initialized in ascending order of value in document sorted order to make it easier for me to unit test.